### DISCUSSION TITLE
Cannot play on dedicated

### DISCUSSION THREAD

**Discussion by Souza-R** [Vdauphin/HeartsAndMinds#1345] [Question and answer]
Yo guys! HM works fine on editor.. everything..

but when i try to load the mission via TADST .. i start as a seagull! 

Map: UMB Colombia

**Comment by Souza-R** [comment]
![case1](https://user-images.githubusercontent.com/87534514/161386953-f0d30c45-031a-40fc-9e61-ba280da5321e.png)
![case2](https://user-images.githubusercontent.com/87534514/161386958-358c5a41-52c4-4bd2-a284-de824cdfdda0.png)

Everything is fine on eden editor.. then when i try it on dedicated.. theres no object.. and i respawn as a seagull

![case3dedicated](https://user-images.githubusercontent.com/87534514/161386962-7461ca24-f258-4ba6-a8ac-852d4c4b04f1.png)

RPT link: https://drive.google.com/file/d/1o_zxIbdSrhHQKMxlPTKzPevWIhHtH_Sk/view?usp=sharing

**Reply by korda2310** [reply]
Hi, did you solve your problem because I have the same thing on a dedicated server also launched with TADST?
